### PR TITLE
Filter tags for ambassadors page

### PR DESCRIPTION
**Pull request by LeoDiduch**

### CODE REVIEW

**Review by BastienSaulnier — APPROVED** [review]
SnE

File: frontend/src/pages/Ambassadors.jsx [hunk 1]
```diff
@@ -3,10 +3,12 @@ import "./styles/Ambassadors.scss";
 import FilterTab from "../components/FilterTab";
 import ProfileCard from "../components/ProfileCard";
 import axios from "axios";
+import { useSelector } from "react-redux";
 const { apiCall } = require("../conf");
 
 function Ambassadors() {
   const [ambassadors, setAmbassadors] = useState([]);
+  const filter = useSelector(state => state.filter);
```
Comment (LoicBrassart) [inline frontend/src/pages/Ambassadors.jsx:11]: (It's a **very** bad practice to name a variable with the name of a JS function)

**Review by LoicBrassart — APPROVED** [review]
SnE

File: frontend/src/pages/Ambassadors.jsx (hunk 1 above)
Comment (BastienSaulnier) [reply frontend/src/pages/Ambassadors.jsx:11]: OK i will fix this the next PR

**Pull request merged**